javascript-carousel/main.js

```diff
@@ -0,0 +1,117 @@
+var $arrowNext = document.querySelector('.arrow-right');
+var $arrowPrevious = document.querySelector('.arrow-left');
+var $imageElement = document.querySelector('img');
+var $allCircles = document.querySelectorAll('.fa-circle');
+var $circleRow = document.querySelector('.progress-dot-row');
+var position = null;
+var imageOneUrl = 'images/001.png';
+var imageTwoUrl = 'images/004.png';
+var imageThreeUrl = 'images/007.png';
+var imageFourUrl = 'images/025.png';
+var imageFiveUrl = 'images/039.png';
```

Comment:
I would store this in an array that you can loop over as necessary to find the next image needed.